### ISSUE TITLE
The types of credit cards that can be scanned with Lens24.

### ISSUE THREAD

**Issue by katol16** [vlasentiy/Lens24#3]
Hello! I tried scanning some credit cards with Lens24, and it works fine for some cards, but not all. For example, I tried using a Revolut card, and it didn’t work as expected. Do you know what types of credit cards work well with Lens24? It would be very helpful to have a list of these types, as I need it for testing purposes.

**Comment by vlasentiy** [comment]
Hi, you can scan Visa and Mastercard with 16 digits (4-4-4-4). Not American Express 15 digits.

**Comment by katol16** [comment]
also tried using a Revolut card, which is a 16-digit Visa, but it didn’t work. I suspect it might be due to the different placement of the card number. Are there any specific requirements regarding card number placement as well?"

**Comment by vlasentiy** [comment]
Yes, the number position is important, so   it will not be scanned(

**Comment by katol16** [comment]
I guess only this placement works fine, right?
![ph-visa-platinum-card-498x280](https://github.com/user-attachments/assets/8055fd31-f4bf-4717-abb7-980436803d9d)

**Comment by vlasentiy** [comment]
Yes

**Comment by Riteshkr79** [comment]
its not detecting 15 digit card ?

Any update or anything you can suggest how to make it detect 15 digit card ?
